examples/daal4py/sycl/sklearn_sycl.py

```diff
@@ -169,7 +169,6 @@ def device_type_to_str(queue):
             devices.append(dpctl.SyclQueue('gpu'))
 
     else:
-        devices.append('host')
```

Comment:
👍

Reply:
suggest to merge this separately as this PR would take time to form new examples and structure
https://github.com/intel/scikit-learn-intelex/pull/1153